apps/src/lab2/views/components/instructions.module.scss

```diff
@@ -182,7 +186,9 @@
   }
 
   code {
-    color: $neutral_light;
+    // Color copied for consistency with bootstrap styles auto-applied to SafeMarkdown component
+    background-color: #f7f7f9;
```

Comment:
is there a reason we don't just use the existing background color for the instructions panel?

Reply:
Just that usually we use a slightly darker background color for code.  I grabbed this value from a Java Lab level, but I believe it's the same slightly-gray color in all other labs too.

Reply:
Interesting.  It must be subtle, because I don't quite see it in the screenshots.  At any rate, maybe worth hoisting into a constant, at least in this file, if not in `color.scss`?

Reply:
I agree that it's subtle. I think it's because in other labs the primary background is pure white, but in the lab2 component it's `$neutral_light`.

Reply:
It was my earlier understanding that `$neutral_light` is the desired background color where we'd formerly use `white`, so Music Lab and associated Lab2 components use it quite a bit.

Given that, it might be worth picking a new background color, or not using one, for code?  When there is only a tiny bit of a difference, it can feel more like a subtle mismatch than a deliberate contrast.

Reply:
Per Mark's suggestion in a Slack DM, I've updated the styles to use teal on white: 
![image (201)](https://github.com/code-dot-org/code-dot-org/assets/43474485/9a33deb4-1a7b-4591-9f6c-48c9d09e63e1)

Let me know if either of you have thoughts/concerns about this. Brendan this also means we're using the existing background now, to your original question.